drivers/Makefile.dep

```diff
@@ -251,3 +251,18 @@ ifneq (,$(filter adc%1c,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
   USEMODULE += adcxx1c
 endif
+
+ifneq (,$(filter winc1500,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio
+  FEATURES_REQUIRED += periph_spi
+  USEMODULE += xtimer
+  USEMODULE += core_mbox
+  # Import WINC1500 API from Atmel
+  USEPKG += winc1500
```

Comment:
I don't like that the package and the module have the same name, while residing in different paths. Can't you move all the `winc5100` stuff to `pkg/`? The `nordic_softdevice_ble` driver is another example of where a driver (including netdev wrapper) was ported in that manner.

Reply:
Not a big deal for me :) I will work on it.

Reply:
They're moved to `pkg/` now :)